### ISSUE TITLE
Boards EB10/EB11 lose skin CAN packets  

### ISSUE THREAD

**Issue by marcoaccame** [robotology/icub-firmware#29]
On purple robot I have experienced that if the skin on lower legs is enabled together with inertial sensors (all CAN frames come from the MTB boards), then there is an overflow of the buffer containing the skin CAN frames, even at 1 ms TX rate.

**Comment by marcoaccame** [comment]
I have solved the problem by a change of policy in parsing the can frames.

Now, all CAN frames from MTB boards are parsed and assigned to the relevant objects. 

If they are about skin data, they are passed to the EOtheSKIN object which:
- if possible fills the status which is regularly broadcasted,
- if the status is full (so far it holds 9 items), it temporarily stores them in an intermediate buffer,
- each time the status is transmitted, skin data is moved from the intermediate buffer into the status, so that it is ready for next transmission.

If they are about inertial sensors, they are passed to EOtheInertial object which already acted as the EOtheSKIN now does.

Tested with purple robot and committed sources and produced a new release for EMS boards (version 2.13). 

Updated purple and blue robots.
